Code/GraphMol/SubstructLibrary/SubstructLibrary.h

```diff
@@ -549,7 +557,8 @@ class RDKIT_SUBSTRUCTLIBRARY_EXPORT SubstructLibrary {
   boost::shared_ptr<ROMol> getMol(unsigned int idx) const {
     // expects implementation to throw IndexError if out of range
     PRECONDITION(mols, "molholder is null in SubstructLibrary");
-    return mols->getMol(idx);
+    const bool sanitize=true;
```

Comment:
Isn't setting `sanitize` to true here changing the current behavior?
There aren't that many circumstances where it's necessary, so I disagree with it being the default.

Reply:
This is in the SubstructLibrary.getMol function which is only called by the user.  I made the decision to always return sanitized molecules when the user calls a function to avoid future bug reports :)